src/style.css

```diff
@@ -1 +1,69 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header__left-side {
+  width: 40px;
+  height: 40px;
+}
+
+.header__left-side .nav__logo {
+  display: inline-block;
+  width: 40px;
+  height: 40px;
+}
+
+.header__right-side .nav .nav__list {
```

Comment:
it is bad idea to have a selector with 3 parts 
later you will have a war of specificity